src/patternfly/components/DataToolbar/examples/DataToolbar.md

```diff
@@ -290,43 +302,45 @@ The `.pf-m-toggle-group` controls when, and at which breakpoint, filters will be
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-m-show{-on-[breakpoint]}` | `.pf-c-data-toolbar__group.pf-m-toggle-group`, `.pf-c-data-toolbar__expandable-content` | Modifies toolbar element visibility at breakpoint. This selector must be applied consistently to toggle group and expandable content. |
-| `.pf-m-toggle-group-container` | `.pf-c-data-toolbar__content` | Modifies the data toolbar content for toggle group, expandable content layout. |
 | `.pf-m-chip-container` | `.pf-c-data-toolbar__content`, `.pf-c-data-toolbar__group` | Modifies the data toolbar element for applied filters layout. |
+| `.pf-m-chip-container` | `.pf-c-data-toolbar__content`-section, `.pf-c-data-toolbar__group` | Modifies the data toolbar element for applied filters layout. |
```

Comment:
this is already on the line above it - do you just need to add `-section` to the `.pf-c-data-toolbar__content` selector?